### PR TITLE
Fixed build instructions for new idf

### PR DESCRIPTION
**Pull request by agdl**
Now we are using idf 3.3 Furthermore for new installations, make menuconfig has to be launched once :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ This firmware uses [Espressif's IDF](https://github.com/espressif/esp-idf)
 
 1. [Download the ESP32 toolchain](http://esp-idf.readthedocs.io/en/v3.1/get-started/index.html#setup-toolchain)
 1. Extract it and add it to your `PATH`: `export PATH=$PATH:<path/to/toolchain>/bin`
-1. Clone **v3.1.3** of the IDF: `git clone --branch v3.1.3 --recursive https://github.com/espressif/esp-idf.git`
+1. Clone **v3.3** of the IDF: `git clone --branch release/v3.3 --recursive https://github.com/espressif/esp-idf.git`
 1. Set the `IDF_PATH` environment variable: `export IDF_PATH=<path/to/idf>`
+1. Run `make menuconfig` then press save and then exit
```
Comment (sandeepmistry) [inline README.md]: @facchinm should we commit a new SDK config to remove this additional step?

Comment (facchinm) [reply README.md]: Absolutely, I wanted to do it but I got stuck in other stuff :)

Comment (sandeepmistry) [reply README.md]: @agdl could you please add the new config to this pull request and remove this new line from the read me. Thanks.

**Comment by sandeepmistry** [comment]
So it seems 3.3 is not officially released yet: https://github.com/espressif/esp-idf/releases

**Comment by sandeepmistry** [comment]
@agdl ping ...

**Comment by sandeepmistry** [comment]
Closing in favour of #42.

**Pull request closed without merging**